run.go

```diff
@@ -108,6 +108,9 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	if spec.Process.Cwd == "" {
 		spec.Process.Cwd = DefaultWorkingDir
 	}
+	if err := os.MkdirAll(filepath.Join(mountPoint, b.Workdir), 0755); err != nil {
+		return fmt.Errorf("error ensuring working directory %q exists: %v)", b.Workdir, err)
```

Comment:
The two hunks in this PR ended up using slightly different errors messages, though it wasn't something I was concerned about at the time.